Docker image fails to build using make

Thanks for the work on this amazing project!

The latest version fails to build the Docker image using `make`:

```shell
➜ make docker
docker image build -t "sslh:heads/master-0-g7e3f723" .
invalid argument "sslh:heads/master-0-g7e3f723" for "-t, --tag" flag: invalid reference format
See 'docker image build --help'.
make: *** [Makefile:123: docker] Error 125
```
I know the `Makefile` sets this environment variable by running a shell script to generate the version number:

```shell
VERSION=$(shell ./genver.sh -r)
```

It looks like `docker image build` chokes on the forward slash (`/`) that's being generated by the version number script. I can build it manually using something like this:

```shell
docker image build -t "sslh:heads-master-0-g7e3f723" .
docker image tag "sslh:heads-master-0-g7e3f723" sslh:latest
```

--- a/genver.sh
+++ b/genver.sh
@@ -10,12 +10,12 @@ fi
 if [ ! -d .git ] || ! `(git status | grep -q "On branch") 2> /dev/null`; then
         # If we don't have git, we can't work out what
         # version this is. It must have been downloaded as a
-        # zip file. 
-        
+        # zip file.
+
         # If downloaded from the release page, the directory
         # has the version number.
         release=`pwd | sed s/.*sslh-// | grep "[[:digit:]]"`
-        
+
         if [ "x$release" = "x" ]; then
             # If downloaded from the head, Github creates the
             # zip file with all files dated from the last
@@ -28,7 +28,7 @@ fi
 if [ -d .git ] && head=`git rev-parse --verify HEAD 2>/dev/null`; then
 	# generate the version info based on the tag
 	release=`(git describe --tags || git --describe || git describe --all --long) \
-		2>/dev/null | tr -d '\n'`
+		2>/dev/null | tr -s '/' '-' | tr -d '\n'`
 
 	# Are there uncommitted changes?
 	git update-index --refresh --unmerged > /dev/null

